src/dune_rules/lib_deps.mli

```diff
@@ -0,0 +1,7 @@
+type t = Lib_dep.t list
+
+val decode :
+     allow_re_export:bool
+  -> (Lib_dep.t list, Dune_lang.Decoder.values) Dune_lang.Decoder.parser
+
+val of_pps : Lib_name.t list -> Lib_dep.t list
```

Comment:
Could you move this under `Lib_dep.L` or does it cause cycles?

Reply:
that's nicer indeed. done in [d4cb726](https://github.com/ocaml/dune/pull/7239/commits/d4cb726879888395b1e9a0b9c77d625333f12ec5)